Students/Alb Gabriel - Marian/Lab03/utcn/ro/Phone.java

```diff
@@ -0,0 +1,36 @@
+package utcn.ro;
+
+public class Phone{
+
+    static int attributeSharingValueForAllInstances;
+    String imeiNumber;
+    String brandName;
+    double displaySize;
+    int volume=0;
+    String contactPersonName;
+
+    //Constructor
+    public Phone()
+    {
+        displaySize = 5.5;
+
+    }
+
+    public Phone(String imeiNumber){
+        this.imeiNumber = imeiNumber;
+        this.powerOn();
+    }
+
+    public void powerOn(){
```

Comment:
spatiu inainte de acolada. Pastreaza acelasi stil de acolade peste tot. La constructor le-ai pus ca si in C